### PR TITLE
fix: update pull image validation form (#2333)

### LINKED ISSUE #2333: Can't copy/paste when pulling an image
### Bug description

When I go to pull an image and paste an image name into the `Image to Pull:` field, it's still invalid. I have to add a character and delete it in order for the form to validate.

### Operating system

macOS 13.3 M1

### Version

next (development version)

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

this PR updates how the form is validated. 

### Screenshot/screencast of this PR

pull-image-ui

### What issues does this PR fix or reference?

it resolves #2333 

### How to test this PR?

1. Images -> pull image -> try to copy/paste an image name/write the image name manually/delete it/enter a white string/whatever... and check that the message + button are shown as expected

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by benoitf — COMMENTED** [review]
do you think we can add a test that do the copy/paste ?

**Comment by benoitf** [comment]
https://testing-library.com/docs/ecosystem-user-event/#pasteelement-text-eventinit-options

File: package.json [hunk 1]
```diff
@@ -74,6 +74,8 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.0.2",
     "@rollup/plugin-typescript": "^11.1.0",
+    "@testing-library/dom": "^9.2.0",
+    "@testing-library/user-event": "^14.4.3",
```
Comment (benoitf) [inline package.json]: there is already one in the packages/render/package.json file

Both need to go in the renderer `package.json` file as they're no usecase of this library on the other part (like main)

Comment (benoitf) [reply package.json]: https://github.com/containers/podman-desktop/blob/main/packages/renderer/package.json#L20

File: package.json [hunk 2]
```diff
@@ -74,6 +74,8 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.0.2",
     "@rollup/plugin-typescript": "^11.1.0",
+    "@testing-library/dom": "^9.2.0",
```
Comment (benoitf) [inline package.json]: and for this one I thought we have https://github.com/containers/podman-desktop/blob/main/packages/renderer/package.json#L18

**Review by benoitf — CHANGES_REQUESTED** [review]
dependencies should be in renderer
I also think we already have required dependencies

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
thanks @lstocchi

**Pull request merged**